src/core/logging/sentry/log.ts

```diff
@@ -8,14 +8,41 @@ const warningLevel: Sentry.SeverityLevel = 'warning';
 const errorLevel: Sentry.SeverityLevel = 'error';
 const fatalLevel: Sentry.SeverityLevel = 'fatal';
 
+export const tagKeys = {
+  CATEGORY: 'CATEGORY',
+  LABEL: 'LABEL',
+} as const;
+
+export const tagCategoryValues = {
+  SERVER: 'SERVER',
+  AUTH: 'AUTH',
+  UI: 'UI',
+  WHITEBOARD: 'WHITEBOARD',
+} as const;
+
+type TagsKeysType = keyof typeof tagKeys;
+type TagType = {
+  [key in TagsKeysType]?: string;
+};
+
+const setTags = (tags: TagType | undefined, scope: Sentry.Scope) => {
```

Comment:
i think it makes more sense the mandatory parameters to be first. Also it doesnt make sense to set tags when there are no tags provided.